test/assets/unrunnableTestCases.csv

```diff
@@ -428,6 +448,7 @@ fold-left-011,AssertionError: expected [Function] to throw an error
 fold-left-013,AssertionError: expected [Function] to throw error matching /XPTY0004/ but got 'Unrecognized type "elementTest".'
 fold-left-014,AssertionError: expected [Function] to throw error matching /XPTY0004/ but got 'Unrecognized type "elementTest".'
 fold-left-016,Error: XPST0008, The variable a is not in scope.
+fold-left-021,RangeError: Maximum call stack size exceeded
```

Comment:
Can we check what causes this? This seems rather dangerous if this is indeed a regression.